utils/pdfpc.typ

```diff
@@ -1,3 +1,54 @@
+#let pdfpc-file = {
+  locate(loc => {
+    let arr = query(<pdfpc>, loc).map(it => it.value)
```

Comment:
```suggestion
  context{
    let arr = query(<pdfpc>).map(it => it.value)
```

Reply:
Typst 0.13.0 removed compatibility behavior of `locate`, `context` should be used instead.
My suggested changes make this code snippet work with newer versions of Typst